src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -254,6 +260,22 @@ public FormValidation doCheckRegion(@QueryParameter String region){
         return ret;
     }
 
+    @RequirePOST
+    public FormValidation doCreateS3Bucket(@QueryParameter String container, @QueryParameter String prefix,
+                                                   @QueryParameter String region, @QueryParameter String credentialsId){
+        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+        FormValidation ret = FormValidation.ok("success");
+        try {
+            S3BlobStore provider = new S3BlobStoreTester(container, prefix, region, credentialsId);
+            provider.createS3Bucket(container);
+        } catch (Throwable t){
+            String msg = processExceptionMessage(t);
+            ret = FormValidation.error(StringUtils.abbreviate(msg, 200));
+            LOGGER.finest(t.getMessage());
```

Comment:
If you are going to log at sub-`INFO` levels, you may as well include the whole stack trace for analysis.